commander/src/bootstrapping/commands/generator/export.ts

```diff
@@ -119,8 +121,16 @@ export abstract class ExportCommand extends BaseIPCClientCommand {
 
 		if (flags.output) {
 			fs.writeJSONSync(flags.output, output, { spaces: ' ', mode: OWNER_READ_WRITE });
+			this.log(`Generator info is exported to ${flags.output}`);
 		} else {
+			// write to cwd
+			fs.writeJSONSync(path.join(process.cwd(), 'genInfo.json'), output, {
+				spaces: ' ',
+				mode: OWNER_READ_WRITE,
+			});
+
 			this.log(JSON.stringify(output, undefined, '  '));
```

Comment:
If the output is now exported to a file, perhaps we don't need to print it to stdout anymore?

Reply:
Yes, It's not necessary anymore to print to stdout.